tonic-types/src/lib.rs

```diff
@@ -1,4 +1,12 @@
 //! A collection of useful protobuf types that can be used with `tonic`.
+//!
+//! This crate also introduces the [`WithErrorDetails`] trait and implements it
+//! in [`tonic::Status`], allowing the implementation of the
+//! [gRPC Richer Error Model] with [`tonic`] in a convenient way.
+//!
+//! [`tonic::Status`]: https://docs.rs/tonic/0.8.0/tonic/struct.Status.html
```

Comment:
I feel like we will forget to update this version number. I think we may want to link to latest instead?

Reply:
Ok, I addressed this in the latest adjustments commit.